doc/jvm/xgboost4j_spark_tutorial.rst

```diff
@@ -153,6 +153,39 @@ Now, we have a DataFrame containing only two columns, "features" which contains
 "sepal length", "sepal width", "petal length" and "petal width" and "classIndex" which has Double-typed
 labels. A DataFrame like this (containing vector-represented features and numeric labels) can be fed to XGBoost4J-Spark's training engine directly.
 
+Dealing with missing values:
+
+In the case that a feature column contains missing values for any reason (could be related to business logic / wrong data ingestion process / etc.), the user should decide on a strategy of how to handle it. 
+The choice of approach depends on the value representing 'missing' which fall into four different categories: 
+(1) 0
+(2) NaN 
+(3) NULL
+(4) any other value which is not mentioned in (1) / (2) / (3)
+
+We introduce the following approaches dealing with missing value and their fitting scenarios:
+(*) Skip VectorAssembler (using setHandleInvalid = "skip") directly. Used in (2), (3).
+(*) Keep it (using setHandleInvalid = "keep"), and set the "missing" parameter in XGBClassifier/XGBRegressor as the value representing missing. Used in (2) and (4).
+(*) Keep it (using setHandleInvalid = "keep") and transform to other irregular values. Used in (3).
+(*) Transform it to other values. Used in (1).
```

Comment:
you can keep your original description about why it is tricky with (1)

Reply:
Never mind, already updated it and the message to the user is basically the same.